src/PIL/ImageFont.py

```diff
@@ -788,8 +788,13 @@ def truetype(font=None, size=10, index=0, encoding="", layout_engine=None):
                      .. versionadded:: 4.2.0
     :return: A font object.
     :exception OSError: If the file could not be read.
+    :exception ValueError: If the size given is invalid.
     """
 
+    if size <= 0:
+        msg = "font size must be positive"
```

Comment:
```suggestion
    :exception ValueError: If the font size is not greater than zero.
    """

    if size <= 0:
        msg = "font size must be greater than 0"
```
Suggestion, to be closer to an existing error message.
https://github.com/python-pillow/Pillow/blob/a1d22970295d1edd48970fff0825a9e48fbfd75e/src/PIL/ImageOps.py#L367